import boto3
from botocore.exceptions import NoCredentialsError

def upload_to_s3(file_name, bucket, s3_file):
    s3 = boto3.client('s3')
    try:
        s3.upload_file(file_name, bucket, s3_file)
        print(f"File {file_name} uploaded to {bucket}/{s3_file}")
    except FileNotFoundError:
        print("The file was not found")
    except NoCredentialsError:
        print("Credentials not available")

if __name__ == "__main__":
    file_name = "path/to/your/file"
    bucket_name = "your-s3-bucket"
    s3_file_name = "backup.tar.gz"
    upload_to_s3(file_name, bucket_name, s3_file_name)
